unison-core/src/Unison/Hashable.hs

```diff
@@ -37,9 +37,9 @@ class Accumulate h where
 accumulateToken :: (Accumulate h, Hashable t) => t -> Token h
 accumulateToken = Hashed . accumulate'
 
-accumulate' :: (Accumulate h, Hashable t) => t -> h
-accumulate' = accumulate . (hashVersion :). tokens
-  where hashVersion = Tag 2
+hash, accumulate' :: (Accumulate h, Hashable t) => t -> h
+accumulate' = accumulate . tokens
+hash = accumulate'
```

Comment:
This is no longer used for our hashing, so we don't need to salt anything.  Perhaps we'll refactor `Parser.hs` at some point to no longer require this module.